Source/AST/AST.h

```diff
@@ -170,7 +170,9 @@ namespace AST
 	struct ErrorRecord
 	{
 		std::string message;
-		ErrorRecord(std::string&& inMessage) : message(std::move(inMessage)) {}
+		uint32 line;
+		uint8 column;
+		ErrorRecord(std::string&& inMessage, uint32 inLine = 0, uint8 inColumn = 0) : message(std::move(inMessage)), line(inLine), column(inColumn) {}
```

Comment:
Can you make `ErrorRecord` use `Core::TextFileLocus` instead of a direct line+column number?